docs/usage-cli.md

```diff
@@ -0,0 +1,208 @@
+# Command-line Interface
+
+## Usage
+
+```
+$ dredd '<API Description Document>' '<API Location>' [OPTIONS]
+```
+
+Example:
+
+```
+$ dredd ./apiary.md 'http://localhost:3000'
+```
+
+## Arguments
+
+### API Description Document (string)
+
+URL or path to the API Description Document (e.g. API Blueprint).  
+**Sample values:** `./apiary.apib`, `http://example.com/apiary.apib`
+
+### API Location (string)
+
+URL, the root address of your API.  
+**Sample values:** `http://localhost:3000`, `http://api.example.com`
+
+## Configuration File
+
+If you use Dredd repeatedly within a single project, the preferred way to run it is to first persist your configuration in a `dredd.yml` file. With the file in place you can then run Dredd every time simply just by:
+
+```
+$ dredd
+```
+
+Dredd offers interactive wizard to setup your `dredd.yml` file:
+
+```
+$ dredd init
+```
+
+See below how sample configuration file could look like. The structure is
+the same as of the [Dredd Class configuration object](usage-js.md#configuration-object-for-dredd-class).
+
+```yaml
+reporter: apiary
+custom:
+  - "apiaryApiKey:yourSecretApiaryAPiKey"
+  - "apiaryApiName:apiName"
+dry-run: null
+hookfiles: "dreddhooks.js"
+sandbox: false
+server: rails server
+server-wait: 3
+init: false
+custom: {}
+names: false
+only: []
+output: []
+header: []
+sorted: false
+user: null
+inline-errors: false
+details: false
+method: []
+level: info
+timestamp: false
+silent: false
+path: []
+blueprint: apiary.apib
+endpoint: "http://localhost:3000"
+```
+
+## CLI Options Reference
+
+Remember you can always list all available arguments by `dredd --help`.
+
+### --details, -d
+Determines whether request/response details are included in passing tests.
+  
+
```

Comment:
Note, this applies to the whole doc. It looks like it goes between one and two lines between these options. Might be nice to have them synced.